### ISSUE TITLE
Training strategy on one 3090ti GPU

### ISSUE THREAD

**Issue by Zeyu1226-mt** [dk-kim/DFWSGAR#12]
I'm sorry to bother you. I'm working with a single 3090ti GPU (24G). Could you kindly advise on the appropriate settings for parameters such as learning rate, epochs, seed，and weight_decay? Any general guidance would be greatly appreciated. Thank you so much.

**Comment by dk-kim** [comment]
Sorry for the late reply. Unfortunately, our experiments were mainly performed on 4 Titan XP GPUs. Nonetheless, we do not think that conducting experiments in other GPU environments will yield a substantial impact, provided that the batch size is adjusted accordingly.

**Comment by Zeyu1226-mt** [comment]
> Sorry for the late reply. Unfortunately, our experiments were mainly performed on 4 Titan XP GPUs. Nonetheless, we do not think that conducting experiments in other GPU environments will yield a substantial impact, provided that the batch size is adjusted accordingly.

ok, i will try, thanks

**Comment by dk-kim** [comment]
Sorry for the confusion. Our experiments were mainly performed on 2 RTX 3090 GPUs.
